src/List/List.js

```diff
@@ -55,6 +55,7 @@ export type Props = {
   /**
    * The component used for the root node.
    * Either a string to use a DOM element or a component.
+   * The default value is a `ul`.
```

Comment:
No need for duplicating the information. You can remove this line:
![capture d ecran 2017-12-11 a 10 12 40](https://user-images.githubusercontent.com/3165635/33823589-efea4da0-de5b-11e7-86aa-a08e031a8f15.png)